plugin/db/util/driverutil.go

```diff
@@ -207,11 +207,21 @@ func BeginMigration(ctx context.Context, executor MigrationExecutor, m *db.Migra
 			return -1, common.Errorf(common.MigrationAlreadyApplied,
 				fmt.Errorf("database %q has already applied version %s", m.Database, m.Version))
 		case db.Pending:
-			return -1, common.Errorf(common.MigrationPending,
-				fmt.Errorf("database %q version %s migration is already in progress", m.Database, m.Version))
+			err := fmt.Errorf("database %q version %s migration is already in progress", m.Database, m.Version)
+			log.Debug(err.Error())
```

Comment:
maybe we should add some context message info.

Reply:
Probably that's alright as we do debug logging.